qiskit/primitives/containers/bit_array.py

```diff
@@ -62,6 +62,19 @@ class BitArray(ShapedMixin):
     This object contains a single, contiguous block of data that represents an array of bitstrings.
     The last axis is over packed bits, the second last axis is over shots, and the preceding axes
     correspond to the shape of the pub that was executed to sample these bits.
+
+    You typically get this object back as one part of a :class:`.DataBin` accessed through
+    a single :class:`.PubResult.data`.  Users do not typically need to create this class themselves.
```

Comment:
I only suggest this because I do find myself creating them my own from time to time.

```suggestion
    a single :class:`.PubResult.data`.  Users do not typically need to create this class themselves, 
    though static methods starting with ``from_`` will help construct them from other formats 
    when required.
```

Reply:
Thanks - done in b00c49acc5.